python.en.md

```diff
@@ -0,0 +1,69 @@
+# Python coding style
+
+## Introduction
+
+This guide defines the conventions for writing Python code at Cookpad Inc.
+
+## General rules:
+
+ - value readability and maintainability
+ - keep it simple
+ - be consistent in your style
+ - for things not in this guide, be consistent with the style of the project you're working on
+
+## Python version
+
+ - **[SHOULD]** When possible, projects should use Python 3 syntax. Use of Python 2 should be motivated by external reasons, like dependencies or heavy use of legacy code.
+
+## Language Features
+
+🙌Avoid:
```

Comment:
I added some "shoulds" and "musts" but some rules are very soft like "OK to use under certain conditions".